### PR TITLE
Updated schema according to restructured dtos

### PR DESCRIPTION
**Pull request by BenasB**
## What was done

Updated schema according to changes in vu-vibedosa/apsitvarkom-be#99

### CODE REVIEW

File: src/types/backEnd/PollutedLocationDTO.test.ts [hunk 1]
```diff
@@ -1,18 +1,18 @@
 import { severityLevels } from "../PollutedLocation";
```
Comment (6ar8nas) [inline src/types/backEnd/PollutedLocationDTO.test.ts:1]: fileName still includes DTO

File: src/types/backEnd/PollutedLocationGetResponse.ts [hunk 2]
```diff
@@ -1,13 +1,12 @@
 import PollutedLocation, { severityLevels } from "../PollutedLocation";
 
-/**
- * Data Transfer Object for @type {PollutedLocation}.
- */
-type PollutedLocationDTO = Partial<{
+type PollutedLocationGetRequest = Partial<{
```
Comment (6ar8nas) [inline src/types/backEnd/PollutedLocationGetResponse.ts]: the fileName is GetResponse, but you define GetRequest here

Comment (6ar8nas) [reply src/types/backEnd/PollutedLocationGetResponse.ts]: I believe we should match the naming from be. We're sending a request with some params and receiving a response. This current naming would contradict that.

**Review by 6ar8nas — CHANGES_REQUESTED** [review]
I believe there is a mix-up with the naming here.
If you believe there is not, let's discuss it using some messaging tool.

File: src/types/backEnd/PollutedLocationGetResponse.ts (hunk 2 above)
Comment (BenasB) [reply src/types/backEnd/PollutedLocationGetResponse.ts]: haha yes sorry, I mixed it up, will change to `PollutedLocationResponse` same as in BE

**Review by 6ar8nas — APPROVED** [review]

**Pull request merged**